SPECS/gcc/gcc.spec

```diff
@@ -329,6 +329,11 @@ LD=ld \
     --enable-plugin \
     --enable-shared \
     --enable-threads=posix \
+%ifarch x86_64
+    --with-arch=x86-64-v3 \
+%else
+    --with-arch=armv8.1-a \
+%endif
```

Comment:
I can't review much regarding these options since I just lack experience but it builds, so that's the minimum bar I'm interested in. Could you add a few lines to the summary explaining why we've chosen these options over others? Or point to some internal task, where that's explained better, so that we have a way to understand such changes when we look at them in the future.

Reply:
Sure, I've added details to the summary